### ISSUE TITLE
INFO: Could not find files for the given pattern(s).

### ISSUE THREAD

**Issue by JackismyShephard** [gradio-app/gradio#9974]
### Describe the bug

I noticed that `INFO: Could not find files for the given pattern(s).` is printed to the console whenever starting a gradio application. What is really strange is that this log message is still printed even when I disable python logging, so it seems the message is not being emitted by python code. It is not too big of a deal but it is a bit annoying. Also, I am not sure whether this message can be safely ignored or not (I assume it can since it is has log level INFO). 

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import logging

import gradio as gr

logging.basicConfig(handlers=[logging.NullHandler()])


def greet(name):
    return "Hello " + name + "!"


with gr.Blocks() as demo:
    name = gr.Textbox(label="Name")
    output = gr.Textbox(label="Output Box")
    greet_btn = gr.Button("Greet")
    greet_btn.click(fn=greet, inputs=name, outputs=output, api_name="greet")

demo.launch()


```

### System Info

```shell
Windows 11
Python 12.6
Gradio 5.6.0
```

### Severity

I can work around it

**Comment by abidlabs** [comment]
Hi @JackismyShephard I can't repro this issue locally, I'm pretty sure this is not gradio-related! See similar issues on Stack Overflow, for example: https://stackoverflow.com/questions/68412439/info-could-not-find-files-for-the-given-patterns-error

(If I'm wrong and you can share in a reproducible environment such as a Colab notebook, please share here and we can reopen this issue!)

**Comment by JackismyShephard** [comment]
@abidlabs The thing is that this logging message only occurs after upgrading to gradio 5.6.0. Running on gradio 4.44.1 I get nothing. That leads me to believe that the issue is at least partially due to some changes you have implemented. Did you try and test this on windows? I see that I am not the only one getting this logging message. For example, see here: https://github.com/gradio-app/gradio/issues/9809

**Comment by abidlabs** [comment]
@yvrjsharma could you see if you can reproduce this issue on Windows?

**Comment by JackismyShephard** [comment]
@abidlabs fyi I have tested now on linux (Ubuntu 22.04) and the INFO message indeed does not show up there.

**Comment by abidlabs** [comment]
Ok reopening while we investigate

**Comment by Asachanel** [comment]
I think it might have something to do with the system, I also get "INFO: Could not find files for the given pattern(s)." when I'm running on Win11 , but in Win10 it won't, and I don't understand why

**Comment by avan06** [comment]
After upgrading to Gradio 5, I encountered the same message on Windows. After debugging, I have identified the cause.  

In `/gradio/utils.py`, within the `def get_node_path()` function, it attempts to retrieve `GRADIO_NODE_PATH` from `os.environ`. However, such a path does not exist. As a result, when it executes the following code, the INFO message is generated.

```py
# On Windows, try using 'where' command
if sys.platform == "win32":
    windows_path = (
        subprocess.check_output(["where", "node"])
        .decode()
        .strip()
        .split("\r\n")[0]
    )
```

> INFO: Could not find files for the given pattern(s).

**Comment by Puneet15556** [comment]
Hi! I can reproduce this on Windows 11 with Python. The root cause is in get_node_path() in gradio/utils.py — when node isn't installed, subprocess.check_output(["where", "node"]) lets the where command's stderr ("INFO: Could not find files for the given pattern(s).") leak to the user's console, since only stdout is captured. The fix is to pass stderr=subprocess.DEVNULL to the subprocess call. I'd like to open a PR for this — happy to include verification on Windows.

**Comment by Puneet15556** [comment]
I've implemented and verified a fix for this on Windows 11.

**Root cause:** In `get_node_path()` (`gradio/utils.py`), when node is not installed, `subprocess.check_output(["where", "node"])` only captures stdout — so the `where` command's stderr output (`INFO: Could not find files for the given pattern(s).`) leaks to the user's console on every app launch.

**Fix:** Pass `stderr=subprocess.DEVNULL` to the `where` (Windows) and `which` (Unix) lookups so the probe stays silent. The existing `CalledProcessError` handling and fallbacks are unchanged.

Verified locally on Windows 11 — without the fix the INFO line appears, with it the console stays clean.

The fix is ready on my fork: https://github.com/Puneet15556/gradio/tree/fix/windows-where-node-stderr

I tried to open a PR but pull requests currently seem restricted to collaborators on this repo. Happy to open it as soon as that's possible — or a maintainer is welcome to pull the branch in the meantime.
